cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +26,19 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with incorrect credentials', () => {
+    signInPage.visit();
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword('incorrectPassword');
+    signInPage.clickSignInBtn();
+
+    signInPage.checkWrong('Login failed!');
+
+    signInPage.typeEmail('nonexistent@example.com');
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
 
+    signInPage.checkWrong('Login failed!');
```

Comment:
Ensure that the `checkWrong` method correctly verifies that the error message 'Login failed!' is displayed for non-existent email attempts. This will confirm that the application correctly handles login attempts with non-existent accounts.